pkg/etc/config.go

```diff
@@ -61,7 +63,26 @@ type JobQueue struct {
 	Namespace         string `env:"SCANNER_JOB_QUEUE_REDIS_NAMESPACE" envDefault:"harbor.scanner.trivy:job-queue"`
 	WorkerConcurrency int    `env:"SCANNER_JOB_QUEUE_WORKER_CONCURRENCY" envDefault:"1"`
 	PoolMaxActive     int    `env:"SCANNER_JOB_QUEUE_REDIS_POOL_MAX_ACTIVE" envDefault:"5"`
-	PoolMaxIdle       int    `end:"SCANNER_JOB_QUEUE_REDIS_POOL_MAX_IDLE" envDefault:"5"`
```

Comment:
This might be a breaking change for some people that depended on setting`SCANNER_JOB_QUEUE_REDIS_POOL_MAX_IDLE` working properly.